makefile

```diff
@@ -0,0 +1,3 @@
+#!make
+
+test: npm run test
```

Comment:
No need for a makefile if you're only using npm commands

Reply:
Yalla @omriavidan